lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -232,6 +246,7 @@ data ApiErrorCode
     | WalletAlreadyExists
     | NoRootKey
     | WrongEncryptionPassphrase
+    | WrongBinaryPayload
```

Comment:
Can we be more precise than "Wrong"? What about MalformedTxPayload?

Reply:
changed to `MalformedTxPayload`